consensus/src/config.rs

```diff
@@ -34,6 +37,34 @@ pub struct Config {
     pub max_round: Round,
 }
 
+#[derive(Clone)]
+pub struct LocalIO<D: Data, DP: DataProvider<D>, FH: FinalizationHandler<D>, UB: Write, UR: Read> {
+    pub data_provider: DP,
+    pub unit_backup: UB,
+    pub unit_reader: UR,
+    pub finalization_handler: FH,
```

Comment:
Nit: it would be nice to have this slightly better grouped, provider and finalization handler could be next to each other.